### PR TITLE
ES TN Fixes for Issue #166

### PR DESCRIPTION
**Pull request by zoobereq**
# What does this PR do ?

The fix does the following:
- Leaves sentence-final periods denormalized in clauses ending in URLs (see [#166](https://github.com/NVIDIA/NeMo-text-processing/issues/166)).  
- Adds support for social media tags (e.g. `@jensen`).  
- Replaces string literals with variables (e.g. `":"` -> `"colon"`) throughout the "ELECTRONIC" class.  

# Before your PR is "Ready for review"
**Pre checks**:
- [x] Have you signed your commits? Use ``git commit -s`` to sign.
- [x] Do all unittests finish successfully before sending PR?
   1) ``pytest`` or (if your machine does not have GPU) ``pytest --cpu`` from the root folder (given you marked your test cases accordingly `@pytest.mark.run_only_on('CPU')`).
   2) Sparrowhawk tests ``bash tools/text_processing_deployment/export_grammars.sh --MODE=test ...``
- [x] If you are adding a new feature: Have you added test cases for both `pytest` and Sparrowhawk [here](tests/nemo_text_processing).
- [x] Have you added ``__init__.py`` for every folder and subfolder, including `data` folder which has .TSV files?
- [x] Have you followed codeQL results and removed unused variables and imports (report is at the bottom of the PR in github review box) ?
- [x] Have you added the correct license header `Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.` to all newly added Python files?
- [x] If you copied [nemo_text_processing/text_normalization/en/graph_utils.py](nemo_text_processing/text_normalization/en/graph_utils.py) your header's second line should be `Copyright 2015 and onwards Google, Inc.`. See an example [here](https://github.com/NVIDIA/NeMo-text-processing/blob/main/nemo_text_processing/text_normalization/en/graph_utils.py#L2).
- [x] Remove import guards (`try import: ... except: ...`) if not already done.
- [x] If you added a new language or a new feature please update the [NeMo documentation](https://github.com/NVIDIA/NeMo/blob/main/docs/source/nlp/text_normalization/wfst/wfst_text_normalization.rst) (lives in different repo).
- [x] Have you added your language support to [tools/text_processing_deployment/pynini_export.py](tools/text_processing_deployment/pynini_export.py).
  

**PR Type**:
- [ ] New Feature
- [x] Bugfix
- [ ] Documentation
- [ ] Test

If you haven't finished some of the above items you can still open "Draft" PR.

### CODE REVIEW

File: nemo_text_processing/text_normalization/es/taggers/electronic.py [hunk 1]
```diff
@@ -36,39 +52,55 @@ def __init__(self, deterministic: bool = True):
...
 
         dot = pynini.accep(".")
-        accepted_common_domains = pynini.union(*common_domains)
-        accepted_symbols = pynini.union(*symbols) - dot
-        accepted_characters = pynini.closure(NEMO_ALPHA | NEMO_DIGIT | accepted_symbols)
-        acceepted_characters_with_dot = pynini.closure(NEMO_ALPHA | NEMO_DIGIT | accepted_symbols | dot)
+
+        symbols = [x[0] for x in load_labels(get_abs_path("data/electronic/symbols.tsv"))]
+        symbols = pynini.union(*symbols)
+        symbols_no_period = pynini.difference(symbols, dot)
```
Comment (tbartley94) [inline nemo_text_processing/text_normalization/es/taggers/electronic.py:58]: Use `full_stop` not `dot` Less ambiguity

Comment (tbartley94) [reply nemo_text_processing/text_normalization/es/taggers/electronic.py:58]: Same for instances of period

File: nemo_text_processing/text_normalization/es/taggers/electronic.py [hunk 2]
```diff
@@ -36,39 +52,55 @@ def __init__(self, deterministic: bool = True):
         super().__init__(name="electronic", kind="classify", deterministic=deterministic)
 
         dot = pynini.accep(".")
```
Comment (tbartley94) [inline nemo_text_processing/text_normalization/es/taggers/electronic.py:54]: Why are you defining dot when you have period from the import?

**Review by tbartley94 — CHANGES_REQUESTED** [review]
Will approve after renaming in electronic

File: nemo_text_processing/text_normalization/es/taggers/electronic.py (hunk 1 above)
Comment (zoobereq) [reply nemo_text_processing/text_normalization/es/taggers/electronic.py:58]: Renamed `dot` to `full_stop` as requested.

`period` is a string literal for `.`

File: nemo_text_processing/text_normalization/es/taggers/electronic.py [hunk 3]
```diff
@@ -48,21 +48,15 @@ class ElectronicFst(GraphFst):
...
 
         full_stop_accep = pynini.accep(".")
         full_stop = "."
 
-        symbols = [
-            x[0] for x in load_labels(get_abs_path("data/electronic/symbols.tsv"))
-        ]
+        symbols = [x[0] for x in load_labels(get_abs_path("data/electronic/symbols.tsv"))]
         symbols = pynini.union(*symbols)
         symbols_no_period = pynini.difference(symbols, full_stop_accep)
```
Comment (tbartley94) [inline nemo_text_processing/text_normalization/es/taggers/electronic.py:58]: `symbols_no_full_stop`

**Review by tbartley94 — APPROVED** [review]
Fix one name and LGTM

File: nemo_text_processing/text_normalization/es/taggers/electronic.py (hunk 3 above)
Comment (zoobereq) [reply nemo_text_processing/text_normalization/es/taggers/electronic.py]: Done.

**Review by ekmb — APPROVED** [review]

**Pull request merged**